helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/SchemaComparisonConfiguration.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility;
+
+
+public class SchemaComparisonConfiguration {
+  /**
+   * behaves like avro &lt;= 1.7.2 - non-string props on fields or types are ignored
+   */
+  public static final SchemaComparisonConfiguration PRE_1_7_3 = new SchemaComparisonConfiguration(
+      true, false, false, false, true, false
+  );
```

Comment:
nit:  any way we can use something like named parameters here(i.e. a builder pattern?). otherwise, this isn't that distinguishable from binary 011100001 😛